spec/models/spree/payment/processing_decorator_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe Spree::Payment::Processing do
+  describe '#close!' do
```

Comment:
absolutely that ^ 

Avoid repetition in specs, as they become a burden later if the code changes in one place, and you have to change 10 tests for that. Should be just 1..